roles/ceph-rgw-loadbalancer/templates/haproxy.cfg.j2

```diff
@@ -9,7 +9,11 @@ global
     group       haproxy
     daemon
     stats socket /var/lib/haproxy/stats
-
+{%- if haproxy_frontend_ssl_certificate %}
+    tune.ssl.default-dh-param {{ haproxy_ssl_dh_param }}
+    ssl-default-bind-ciphers {% for cipher in haproxy_ssl_ciphers %}{% if loop.last %}{{ cipher }}{% else %}{{ cipher }}:{% endif %}{% endfor %}
+    ssl-default-bind-options{% for option in haproxy_ssl_options %} {{ option }}{% endfor %}
```

Comment:
To avoid doing a loop and missing the \n you should use the join filter.

```yaml
    ssl-default-bind-ciphers {{ haproxy_ssl_ciphers | join(':') }}
    ssl-default-bind-options {{ haproxy_ssl_options | join(' ') }}
```

Because the current code generates something like:

```console
    tune.ssl.default-dh-param 4096
    ssl-default-bind-ciphers EECDH+AESGCM:EDH+AESGCM    ssl-default-bind-options no-sslv3 no-tlsv10 no-tlsv11 no-tls-tickets
```

Reply:
Thank you, changes have been made :)